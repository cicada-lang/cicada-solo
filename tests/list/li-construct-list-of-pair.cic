the(
  List((String) * String),
  li! [
    the((String) * String, cons("cooked", "cooked")),
    the((String) * String, cons("eaten", "eaten")),
    the((String) * String, cons("tried-cleaning", "tried-cleaning")),
    the((String) * String, cons("understood", "understood")),
    the((String) * String, cons("slept", "slept")),
  ])
