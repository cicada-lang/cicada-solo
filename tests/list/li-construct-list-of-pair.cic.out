li(cons("cooked", "cooked"), li(cons("eaten", "eaten"), li(cons("tried-cleaning", "tried-cleaning"), li(cons("understood", "understood"), li(cons("slept", "slept"), nil))))): List((_: String) * String)